.zuul.playbooks/playbooks/wheel/roles/build-wheel-manylinux/files/build-wheels.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+# Compile wheels
+
+cd /io
+mkdir wheelhouse.final
+
+for PYBIN in /opt/python/*/bin; do
```

Comment:
Let's have this just take a var that we can change based on what we support in the future. So for now we'd have the var have the value: `cp35-cp35m`

Reply:
Done